parser/mysql/const.go

```diff
@@ -167,6 +167,7 @@ const (
 	AuthNativePassword      = "mysql_native_password" // #nosec G101
 	AuthCachingSha2Password = "caching_sha2_password" // #nosec G101
 	AuthSocket              = "auth_socket"
+	AuthTiDBSessionToken    = "tidb_session_token"
```

Comment:
I like the `tidb_` prefix here. That helps to avoids name collisions from future MySQL authentication methods.